applications/jupyter/main.tf

```diff
@@ -137,8 +137,7 @@ module "jupyterhub" {
   k8s_backend_config_name  = var.k8s_backend_config_name
   k8s_backend_service_name = var.k8s_backend_service_name
   k8s_backend_service_port = var.k8s_backend_service_port
-  url_domain_addr          = var.url_domain_addr
-  url_domain_name          = var.url_domain_name
-  members_allowlist        = var.members_allowlist
+  domain                   = var.domain
+  members_allowlist        = split(",", var.members_allowlist)
```

Comment:
Does this PR also fix the issue with the allowlist or is that tracked separately? @blackzlq @umeshkumhar

Reply:
This PR I think is trying to fix that. But I think we should use https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/iap_web_backend_service_iam

Reply:
yeah will take in separate PR.. dont want to stretch this one.